src/types.tsx

```diff
@@ -0,0 +1,6 @@
+type Attempt = {
+  x: number
+  y: string
+}
+
+export default Attempt
```

Comment:
I would not make this the default export. You'll almost certainly have more types you want to add here so I'd export them all as named exports

Reply:
changed it to just export the type declaration